consensus/istanbul/backend/engine.go

```diff
@@ -460,6 +460,18 @@ func (sb *backend) Finalize(chain consensus.ChainReader, header *types.Header, s
 		return nil, err
 	}
 
+	// mimic legacy reward config cache.
+	// if blockNum is multiple of epoch, don't use ParamsAt(blockNum), but use ParamsAt(blockNum - epoch) instead
+	// see https://github.com/klaytn/klaytn/blob/eabdabed10f7c57ee8809f93dbc0ff67ae9f26bf/reward/reward_config_cache.go#L72-L77
```

Comment:
Please link a named version like v1.9.1.

Reply:
fixed in 57e4716b